src/developer_config.yaml

```diff
@@ -9,8 +9,8 @@ global:
   output_auto_outliers: False
   output_outlier_qa : False
   output_estimation_qa: False
-  output_short_form: False
-  dev_test : False
+  output_short_form: True
+  dev_test : True
```

Comment:
I think this has been duplicated as part of #85 - should be easy enough to sort when we come to resolve conflicts

Reply:
That's okay, it will get resolved in rebases